test/api/openSaucedApi.test.ts

```diff
@@ -14,9 +14,9 @@ describe("openSaucedUserEndpoint", () => {
         expect(data.login).toBe("bdougie");
     })
 
-    it("should return a 404 error for a user that does not exist", async () => {
-        const response = await fetch(`${OPEN_SAUCED_USERS_ENDPOINT}/1`);
-        expect(response.status).toBe(404);
+    it("should not return a 200 error for a user that does not exist", async () => {
+        const response = await fetch(`${OPEN_SAUCED_USERS_ENDPOINT}/foolala`);
+        expect(response.status).not.toBe(200);
```

Comment:
refactor (blocking): the API returns an HTTP 400 when a user does not exist, so test for that.

```suggestion
    it("should return a bad request for a user that does not exist", async () => {
        const response = await fetch(`${OPEN_SAUCED_USERS_ENDPOINT}/foolala`);
        expect(response.status).toBe(400);
    })
```